db/schema.rb

```diff
@@ -161,8 +161,8 @@
     t.boolean  "is_missing"
     t.string   "program_country"
     t.string   "timezone"
-    t.integer  "edition_id"
```

Comment:
you run `rake db:migrate` and didn't checkout db/schema so even though you didn't add any migration db/schema was changed in your branch ;)